glusterd2/commands/volumes/grouped-options.go

```diff
@@ -6,27 +6,75 @@ import (
 
 // defaultGroupOptions maps from a profile name to a set of options
 var defaultGroupOptions = map[string]*api.OptionGroup{
-	"profile.gluster-block": {Name: "profile.gluster-block", Options: []api.VolumeOption{{Name: "performance/quick-read", OnValue: "off"},
-		{Name: "performance/read-ahead", OnValue: "off"},
-		{Name: "performance/io-cache", OnValue: "off"},
-		{Name: "performance/md-cache", OnValue: "off"},
-		{Name: "performance/open-behind", OnValue: "off"},
-		{Name: "performance/readdir-ahead", OnValue: "off"},
-		{Name: "performance/write-behind.strict-O_DIRECT", OnValue: "on"},
-		{Name: "protocol/client.filter-O_DIRECT", OnValue: "disable"},
-		{Name: "cluster/replicate.eager-lock", OnValue: "disable"},
-		{Name: "cluster/replicate.quorum-type", OnValue: "auto"},
-		{Name: "cluster/replicate.data-self-heal-algorithm", OnValue: "full"},
-		{Name: "cluster/replicate.locking-scheme", OnValue: "granular"},
-		{Name: "cluster/replicate.shd-max-threads", OnValue: "8"},
-		{Name: "cluster/replicate.shd-wait-qlength", OnValue: "10000"},
-		{Name: "features/shard", OnValue: "on"},
-		{Name: "features/shard.shard-block-size", OnValue: "64MB"},
-		{Name: "user.cifs", OnValue: "off"},
-		{Name: "protocol/server.rpc-auth-allow-insecure", OnValue: "on"}},
-		Description: "Enable this profile for optimal results, in block use case"},
-	"profile.SMB-small-file": {Name: "profile.SMB-small-file",
-		Options: []api.VolumeOption{{Name: "features/upcall.cache-invalidation", OnValue: "on"},
+	"profile.default.replicate": {
+		"profile.default.replicate",
```

Comment:
we may get  `go vet` issues here, please check 
 please check https://github.com/gluster/glusterd2/pull/1377

Reply:
Sorry.. the changes made in your PR got reversed while rebasing. Fixed now